website/docs/d/pi_catalog_images.html.markdown

```diff
@@ -19,13 +19,12 @@ data "ibm_pi_catalog_images" "ds_images" {
 ```
 
 **Notes**
-* Please find [supported Regions](https://cloud.ibm.com/apidocs/power-cloud#endpoint) for endpoints.
-* If a Power cloud instance is provisioned at `lon04`, The provider level attributes should be as follows:
-  * `region` - `lon`
-  * `zone` - `lon04`
-  
-  Example usage:
+- Please find [supported Regions](https://cloud.ibm.com/apidocs/power-cloud#endpoint) for endpoints.
```

Comment:
I do not think making the change from `-` to `*` makes any difference.

Reply:
It doesn't but it is inconsistent between files.